core/miner/storage_fsm/impl/sealing_impl.cpp

```diff
@@ -1132,15 +1132,15 @@ namespace fc::mining {
       return outcome::success();
     }
 
-    auto maybe_precommit_params = getPreCommitParams(info);
+    const auto maybe_precommit_params = getPreCommitParams(info);
     if (maybe_precommit_params.has_error()) {
       logger_->error("PreCommitting sector {} params: {:#}",
                      info->sector_number,
                      maybe_precommit_params.error().message());
       FSM_SEND(info, SealingEvent::kSectorChainPreCommitFailed);
       return outcome::success();
     }
-    auto &precommit_params{maybe_precommit_params.value()};
+    const auto &precommit_params{maybe_precommit_params.value()};
```

Comment:
`precommit_params` is moved on line 1179. Remove const or remove move since constness prevents moving.